src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -313,36 +322,41 @@
         - name: --parameters
           short-summary: Supply deployment parameter values.
           long-summary: >
-            Parameters may be supplied from a parameters file, raw JSON (which can be loaded using `@<file path>` syntax, or as <KEY=VALUE> pairs. Parameters are evaluated in order, so when a value is assigned twice, the latter value will be used.
-            It is recommended that you supply your parameters file first, and then override selectively using KEY=VALUE syntax (example: --parameters params.json --parameters location=westus)
+            Parameters may be supplied from a file using the '@<path>' syntax, a JSON string, or as <KEY=VALUE> pairs. Parameters are evaluated in order, so when a value is assigned twice, the latter value will be used.
```

Comment:
Recommend replacing '@<path>' with '@{file}' for consistency with those other instances (or, since path might be better, change all of those to '@{path}'.